File: 백준/Bronze/1546. 평균/평균.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        
        // 입력 받기
        int N = scanner.nextInt();
        int[] scores = new int[N];
        
        for (int i = 0; i < N; i++) {
            scores[i] = scanner.nextInt();
        }
        
        // 최댓값 찾기
        int maxScore = 0;
        for (int score : scores) {
            if (score > maxScore) {
                maxScore = score;
            }
        }
        
        // 새로운 점수 계산 및 평균 구하기
        double sum = 0;
        for (int score : scores) {
            double newScore = (double) score / maxScore * 100;
            sum += newScore;
        }
        
        double average = sum / N;
        
        // 결과 출력
        System.out.printf("%.2f", average);
    }
}

File: 백준/Bronze/1546. 평균/평균.py
# 입력 받기
N = int(input())
scores = list(map(int, input().split()))
    
# 최댓값 찾기
max_score = max(scores)
    
# 새로운 점수 계산 및 평균 구하기
new_scores = [(score / max_score) * 100 for score in scores]
average = sum(new_scores) / N
    
# 결과 출력 (소수 둘째 자리까지)
print(f"{average:.2f}")